files/docker-daemon.json

```diff
@@ -3,5 +3,6 @@
   "log-opts": {
     "max-size": "10m",
     "max-file": "10"
-  }
+  },
+  "no-new-privileges": true
```

Comment:
Sure this is the right change?  I would have expected `"live-restore": true`

`no_new_privs` might be a good idea anyway, but maybe a different issue?

Reply:
ugh, my bad.  Let me fix

Reply:
@mogren i've put in the correct change and rebased.
Sorry for the confusion :(